### ISSUE TITLE
Picker css class conflicts with Google Drive Picker

### ISSUE THREAD

**Issue by poscar** [amsul/pickadate.js#619]
The css class 'picker' used by pickadate.js conflcits with the picker class used by the Google Drive Picker (https://developers.google.com/picker).

This class should be customizable when configuring the plugin, randomly generated, or more unique (e.g. a prefix) so that it doesn't conflict with other plugins.

**Comment by marcucio** [comment]
+1  maybe we change .picker to be .pad-picker or .pickadatepicker?

**Comment by JeanCarriere** [comment]
+1

**Comment by youssmak** [comment]
+2

**Comment by JeanCarriere** [comment]
Do you think we could get an update on this ?

**Comment by felipemarin** [comment]
this worked for me

.picker-dialog-content {
    position: relative;
}

**Comment by erata** [comment]
@felipemarin thanks. it worked me too.

**Comment by brendon** [comment]
That works but I think it should be fixed in the next release perhaps? @amsul, would you like me to prepare a PR? I guess it's quite a major change, so perhaps we need to release a new major release?

**Comment by DanielRuf** [comment]
In most cases people can adjust the styles and use scoped styles.

Not sure if we should change this as I did not see more issues and comments related to this.

**Comment by amsul** [comment]
I would agree with @DanielRuf. This can easily be solved with scoped styles - plus it'd be a major breaking change for anyone using the project with custom styles.

Maybe it would be better to add an option that appends a custom prefix to the class names (although the CSS names would somehow need to get prefixed accordingly as well)

**Comment by brendon** [comment]
@DanielRuf, are you referring to adjusting Pickadate's styles? Could you elaborate on this a bit?

The issue did cause me a fair bit of confusion with regards to the google file picker as suddenly it just showed up as a little dot on the screen and it wasn't obvious that Pickadate's style overlap was the cause. Only a random (lucky) google search came up with someone that had a similar problem and had asked about an unrelated error message emitted by google picker.
